source/fundamentals/enterprise-auth.txt

```diff
@@ -1,27 +1,192 @@
-Edition.
+Overview
+--------
+
+In this guide, you can learn how to authenticate in MongoDB using the
+**authentication mechanisms** available in the MongoDB Enterprise Edition.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
 
-You can use the following mechanisms with the latest version of the MongoDB
-Enterprise Edition:
+You can use the following authentication mechanisms with the latest version of
+the MongoDB Enterprise Edition:
 
-Kerberos (GSSAPI)
-LDAP (PLAIN)
+- :ref:`GSSAPI/Kerberos <golang-kerberos>`
+- :ref:`LDAP (Plain) <golang-LDAP>`
 
-To authenticate using another mechanism, see
-:ref:`golang-authentication-mechanisms`. To learn more about 
-establishing a connection to your MongoDB cluster, see 
+To authenticate using another mechanism, see the
+:ref:`<golang-authentication-mechanisms>` fundamentals page. To learn
+more about establishing a connection to your MongoDB cluster, see the
 :ref:`golang-connection-guide`.
 
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using the following methods:
+
+- Using a **connection URI**. To learn more about using a connection
+  URI for enterprise authentication, see the :manual:`Server manual
+  entry on Connection String URIs </reference/connection-string/#authentication-options>`.
+
+- Specifying credentials and an authentication mechanism in the
+  ``Credential`` type from the {+driver-short+}. You can then use your
+  ``Credential`` struct to provide options for the ``Client``.
+
+This guide provides information on using types and methods provided by
+the {+driver-short+} for enterprise authentication, rather than using a
+connection URI.
+
+.. _golang-kerberos:
+
+Authenticate to GSSAPI/Kerberos
+-------------------------------
+
+The Generic Security Services API (GSSAPI) authentication mechanism allows the 
+user to authenticate to a Kerberos service using the user's principal.
+
+You must use the "gssapi" `build tag
+<https://www.programming-books.io/essential/go/conditional-compilation-with-build-tags-d1980344374d45c082c914c2aafa50cf>`__
+and specify `cgo support <https://pkg.go.dev/cmd/cgo>`__ during
+compilation to use Kerberos authentication. The default service name for
+Kerberos is "mongodb".
+
+The :ref:`example <golang-kerberos-example>` specifies the authentication mechanism using the following
+placeholders:
+
+- ``username``: Your Kerberos principal. A sample username is ``myuser@KERBEROS.EXAMPLE.COM``.
+- ``password``: Your Kerberos user's password.
+- ``hostname``: The network address of your MongoDB server, accessible by your client.
+- ``port``: Your MongoDB server's port number.
+
+.. tip::
+
+   You don't need to define a password in your ``Credential`` struct if
+   you store authentication keys in ``keytab`` files. You can initialize
+   a credential cache for authenticating the user principal using the
```

Comment:
Question: should this be Kerberos principal per Benji's response, or is "user principal" the correct term here?